app/src/organisms/GripperWizardFlows/ExitConfirmation.tsx

```diff
@@ -35,15 +50,32 @@ export function ExitConfirmation(props: ExitConfirmationProps): JSX.Element {
       subHeader={t('are_you_sure_exit', { flow: flowTitle })}
       isSuccess={false}
     >
-      <SecondaryButton onClick={handleGoBack} marginRight={SPACING.spacing4}>
-        {t('shared:go_back')}
-      </SecondaryButton>
-      <AlertPrimaryButton
-        textTransform={TEXT_TRANSFORM_CAPITALIZE}
-        onClick={handleExit}
-      >
-        {t('shared:exit')}
-      </AlertPrimaryButton>
+      {isOnDevice ? (
+        <Flex justifyContent={JUSTIFY_FLEX_END} gridGap={SPACING.spacing4}>
```

Comment:
![Screenshot 2023-07-07 at 4 53 57 PM](https://github.com/Opentrons/opentrons/assets/474225/84c276e5-9a32-48ce-8095-49f38feffb7a)

JUSTIFY_FLEX_END makes sense but without 100%, it doesn't work well.
![Screenshot 2023-07-07 at 5 01 19 PM](https://github.com/Opentrons/opentrons/assets/474225/436bf4bd-591d-4e23-951d-46f1ad2945c6)


![Screenshot 2023-07-07 at 5 00 22 PM](https://github.com/Opentrons/opentrons/assets/474225/429550e9-8c87-4b9e-bddc-ce0e981b9b05)


```suggestion
        <Flex width="100%" justifyContent={JUSTIFY_FLEX_END} gridGap={SPACING.spacing4}>
```